phoenix-core-client/src/main/java/org/apache/phoenix/expression/util/bson/CommonComparisonExpressionUtils.java

```diff
@@ -0,0 +1,271 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.phoenix.expression.util.bson;
+
+import org.bson.BsonArray;
+import org.bson.BsonBinary;
+import org.bson.BsonDateTime;
+import org.bson.BsonDocument;
+import org.bson.BsonNumber;
+import org.bson.BsonString;
+import org.bson.BsonValue;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.apache.hadoop.hbase.util.Bytes;
+
+/**
+ * Common Util functions to help retrieve BSON Document values based on the given field expressions.
+ */
+public class CommonComparisonExpressionUtils {
```

Comment:
this seems to have comparison and other helper functions - May be BsonUtils class?

Reply:
I can update the class name, but the package name itself has bson in it `org.apache.phoenix.expression.util.bson`, so all classes under the package refers to Bson document update/access utility classes.

Reply:
After looking at other files I see there is a reason you named it this way.